src/dvclive/live.py

```diff
@@ -191,6 +198,78 @@ def log_plot(self, name, labels, predictions, **kwargs):
         data.dump(val, self._step, **kwargs)
         logger.debug(f"Logged {name}")
 
+    def log_params(
+        self, params: Dict[str, ParamsLike], name: str = PARAMS_DEFAULT_NAME
+    ):
```

Comment:
(Product) I am not sure about including this one (at least until we also include equivalents for other data types). 

(Implementation) Regardless, is there a reason for this method to don't reuse `log_param` internally?

Reply:
I think it's okay to start here because we have to start somewhere and params are maybe the most likely scenario for having to log many values at once.

However, can we collapse into one method that accepts either? It's messier to implement but less for users to remember.

Reply:
> (Implementation) Is there a reason for this method to don't reuse log_param internally?

These were slighty different initially, but the ended up converging, I'll change `log_param` so that it uses `log_params` internally.

Reply:
> However, can we collapse into one method that accepts either? It's messier to implement but less for users to remember.

Wouldn't it be confusing? As in: with a single method, if logging a `dict`, are you logging a single parameter that takes a `dict` value (this is somehow related to your nested params comment) or are you logging a group of parameters? Maybe it would be just enough to provide an optional `name` argument to distinguish between the two cases, although I stil feel naming would be odd.

Something like this, perhaps: 
```python
dvclive.log_param(name="optimizer_kwargs", {"learning_rate": 1e-3, "name": "adam"}) # this logs a dict parameter
dvclive.log_param({"learning_rate": 1e-3, "optimizer_name": "adam"}) # this logs a dict parameter
```

Reply:
Any thoughts here?

Reply:
LGTM with `log_param` and `log_params`, thanks!

Reply:
Still seems a little odd because of how similar they both are now, but it's probably the most familiar approach for both dvclive and mlflow users, and it gives the most flexibility to users, so I'm good with it.